examples/examples/attrs-args.rs

```diff
@@ -27,9 +27,9 @@ fn fibonacci_seq(to: u64) -> Vec<u64> {
 }
 
 fn main() {
-    use tracing_subscriber::fmt;
+    use tracing_subscriber::{fmt, EnvFilter};
     let subscriber = fmt::Subscriber::builder()
-        .with_env_filter("attrs_args=trace")
+        .with_env_filter(EnvFilter::try_new("attrs_basic=trace").unwrap())
```

Comment:
can we back out this unrelated change to the examples? (sorry, I think this is my fault)

Reply:
no worries, removed.